frontend/src/__test__/App.test.tsx

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+/* eslint-disable react/react-in-jsx-scope */
```

Comment:
Maybe for react-in-jsx-scope we could disable it for all test files (*.test.tsx)?